Source/i_video.c

```diff
@@ -1208,6 +1208,16 @@ static void I_InitGraphicsMode(void)
                                SDL_TEXTUREACCESS_STREAMING,
                                v_w, v_h);
 
+   // Workaround for SDL 2.0.14 alt-tab bug (taken from Doom Retro)
+#if defined(_WIN32)
+   SDL_version ver;
```

Comment:
This whole new paragraph needs to be enclosed in `{...}` with indentation please, to avoid declaring new variables within code.